site/docs/guides/contributor/release-process/egeria-react-ui-release.md

```diff
@@ -0,0 +1,57 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the Egeria project 2020. -->
+
+### Egeria Ecosystem UI (React UI) Release process
+
+#### Obtaining releases / artifacts for  Ecosystem User Interface (Egeria React UI)
+
+| Location | Usage |
+|---|---|
+| [Maven Central Egeria React UI Release:material-dock-window:](https://mvnrepository.com/artifact/org.odpi.egeria-react-ui){ target=maven } | typically used by other developers integrating with our code |
+| [Github Egeria React UI Release :material-github:](https://github.com/odpi/egeria-react-ui/releases){ target=gh } | source code in `zip` and `tar.gz` formats |
+| `git` | `git checkout Vx.y` to get version as-shipped (each release is tagged at the point it is shipped) |
+| `git` | `git checkout Vx.y` to get version as-shipped (each release is tagged at the point it is shipped) |
+
+
+??? success "1. Agree schedule"
+    --8<-- "docs/guides/contributor/release-steps/agree-schedule.md"
+??? success "2. Track remaining issues and PRs" 
+    --8<-- "docs/guides/contributor/release-steps/track-outstanding.md"
+??? success "3. Create branch"
+    - Checkout master `git checkout main`
+    - Ensure local update `git pull upstream main`
+    - Create branch `git branch release-x.y`
+    - Push to upstream `git push upstream release-x.y`
```

Comment:
As with egeria you need the checkout before the push

Reply:
I am not sure why - if I have updated main to be up to date and then create a branch - the branch will be the same level I think.

Reply:
You are correct there is no need. Though in fact this makes the 'checkout' pointless too, since you could just push your main upstream (which is in fact what line 24 is doing, since the branch you created isn't checked out).

I do tend to do the branch then push. I used to update the version at this point

I don't think it matters which of these 3 variations we have in the doc - perhaps both or neither is clearer